README.md

```diff
@@ -23,6 +26,20 @@ See our [Contribution Guidelines](CONTRIBUTING.md)
 
 Building an MVP with base functionality
 
+* Sign up flow (for new users withoug account)
+* Select an instance [joinmastodon servers](https://joinmastodon.org/de/servers)
+* Sign in with credentials
+* Browse timeline
+* Create a Toot!
+
+## Verifications
+
+### Detekt
```

Comment:
Should we move this section also into here: 
https://github.com/AndroidDev-social/MastodonCompose/blob/main/CONTRIBUTING.md#-build

Reply:
It definitely should. I didn't want to add it straight away because Detekt already fails with some existing issues that should be resolved first before forcing every PR to pass the check (which will obviously fail until they are fixed)